packages/payment-integration-api/src/index.ts

```diff
@@ -138,6 +138,7 @@ export {
 } from './payment';
 export { default as PaymentIntegrationSelectors } from './payment-integration-selectors';
 export { default as PaymentIntegrationService } from './payment-integration-service';
+export { PaymentProviderAuthentication } from './payment-provider-authentication';
 export {
     BraintreeAcceleratedCheckoutCustomer,
     PaymentProviderCustomer,
```

Comment:
Just out of curiosity can we not use `PaymentProviderCustomer` here  for the authentication flow? 

cc @serhii-tkachenko

Reply:
As far as I see Sebastian is not using PaymentProviderCustomer for Authentication flow. May I ask you @animesh1987 to describe what do you mean by that?

Reply:
@animesh1987  Can you please clarify what you mean?

Reply:
I meant to say was that can we try and use the `PaymentProviderCustomer` for the flow, since now we are going to be having two different flows for different payment providers? Which makes checks in clients difficult, since we know at a given time we are only going to support one accelerated checkout solution.

Reply:
@animesh1987 I used PaymentProviderCustomer in this commit https://github.com/bigcommerce/checkout-sdk-js/pull/2193/commits/d40f1eda20441b3c0f3aea9f0a8c4146f0162838 
Can you please take a look? It is is ok, than this PR will be closed and I will add necessary changes in PR PI-517

Reply:
Thanks @bc-sebastianszafraniec looks ok to me at a glance. I was expecting something similar as well. 

@serhii-tkachenko can you also take a look please.

Reply:
Yes, we talked about this solution with @bc-sebastianszafraniec in DM. I like the way how it looks.

Reply:
I am closing this PR. Necessary changes will be added here https://github.com/bigcommerce/checkout-sdk-js/pull/2193